### DISCUSSION TITLE
School not added in github education ( Army public school, Gwalior )

### DISCUSSION THREAD

**Discussion by Viceroycodes** [community/community#38976] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

Hey there! 
My name is Deepak Singh. I'm from class 10 in Army public school, Gwalior.

I want to need this student education pack in my school also. So, please help us to add our school in github education student pack...

[ Army public school, Gwalior ]

Thanks
-Deepak Singh
X-D

**Comment by iamcjwilliams** [comment]
@Viceroycodes Please visit [here](https://education.github.com/discount_requests/pack_application) to be verified
